### DISCUSSION TITLE
Local file/directory evasion?

### DISCUSSION THREAD

**Discussion by NullPlane** [jj-vcs/jj#4213] [Q&A] [answered]
Does jujutsu work with or is there an equivalent to 
`.git/info/exclude`? I've only found `.gitignore` described
for ignored files but I usually add loads of folders n stuff to
my working copy which I do not want to ever see being 
touched by the VCS...

**Comment by yuja** [comment, marked as answer]
`.git/info/exclude` is respected.
https://martinvonz.github.io/jj/latest/git-compatibility/
